docs/index.md

```diff
@@ -206,6 +224,9 @@ See [the list of datetime placeholders](https://docs.python.org/3/library/dateti
 
 - When `paging` in *category settings* is set to `false`, if `size` is not set, all posts will be displayed on the first page; otherwise the first
 `size` posts will be displayed and *the rest will not*.
+- `exclude_index` in *global settings* is used to exclude the index page from the blog collection. This is useful when you want to use the index page as a landing page for your blog.
+- `use_root_dirs` in *global settings* is used to automatically create blog pages for each root directory. Theses pages will be named after the directory name.
```

Comment:
We should also clarify here (as you implemented this way):
- the configuration to be used is from the global settings
- specifying the category with the same name as one from the root directories will override it

Reply:
Hello! What do you mean by "configuration" ? Which configuration ?
I'm sorry I was in two weeks holiday and I'm little lost now D:

Reply:
Sorry, it seems that those lines are mistakenly highlighted... I mean the line below:

> `config_filename` in *global settings* is used to define the filename of the configuration file. See the [category specific options](#category-specific-settings) section for more details.